packages/@icon-magic/config-reader/src/config.ts

```diff
@@ -0,0 +1,102 @@
+import * as path from 'path';
+import * as glob from 'glob';
+import { loadConfigFile } from './config-loader';
+import { validateConfigSchema } from './config-validator';
+import { Icon } from '@icon-magic/icon-models';
+import { IconSetHashInterface } from '@icon-magic/icon-models/src/interface';
```

Comment:
Ideally we don't reach in to internals with imports like this and instead expose all required interfaces top level so you can have a single import per module using a well-defined API footprint. All files internal to a module like this should be considered private.